README.md

```diff
@@ -1,73 +1,53 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome Book 
 
-> Description the project.
+> Plain JavaScript with objects
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html 
+- Javascript
+- CSS
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://yohannesdagnachew.github.io/Awesombooks-ES6/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
 
+Prerequisites: 
+   - Signup/login to your github account and go to https://github.com/yohannesdagnachew/Awsome-Book/.
+   
+Setup
+   - Clone the repository on github to your local terminal
+Install
+   - Ensure you have git installed on your terminal
+  
+Usage
+   - Open the index.html file with your browser to view the webpage
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/yohannesdagnachew)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yohannes-dagnachew-5b163a236)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/yohannesdagnachew/Awesombooks-ES6/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
 ## 📝 License
 
```

Comment:
- [OPTIONAL] This project is supposed to the MIT licensed, please do update your readme file with the license information. Checkout this [article](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) on how to go about it.